lib/auth/oauth_gems.rb

```diff
@@ -226,5 +226,10 @@ def authorize_params
       end
     end
   end
+
+  def callback_url
+    options[:redirect_uri] || (full_host + script_name + callback_path)
```

Comment:
> nitpick, should call super instead of (full_host + script_name + callback_path)

not working... just tested with my branch https://github.com/gdong42/discourse-chinese-localization-pack/tree/temp
not sure if I'm using `super` in the correct way, though - I'm quite new to Ruby
I guess it might be due to the `super callback_url` was removed in the [change](https://github.com/intridea/omniauth-oauth2/commit/26152673224aca5c3e918bcc83075dbb0659717f) that the pull request you posted was trying to add back